translations/frontend/en.json

```
@@ -2429,6 +2429,10 @@
                                 "title": "Entity not tracked"
                             }
                         }
+                    },
+                    "compare":{
+                        "info": "You are comparing the period {start} with period {end}"
```

```suggestion
                        "info": "You are comparing the period {start} with {end}"
```
Just an idea. Doesn't change the meaning and saves some screen space.